src/modules/dashboard/components/ActionsPage/VoteWidget/VoteDetails.tsx

```diff
@@ -152,12 +160,54 @@ const VoteDetails = ({
               </div>
             </div>
             <div className={styles.value}>
-              {voterReward?.motionVoterReward && (
-                <Numeral
-                  unit={getTokenDecimalsWithFallback(nativeToken?.decimals)}
-                  value={voterReward.motionVoterReward}
-                  suffix={` ${nativeToken?.symbol}`}
-                />
+              {voterReward?.motionVoterReward && !loadingVoterReward && (
```

Comment:
Please don't put the `loading` variable into this check.

Add a loading state to this component, using the `MiniSpinnerLoader` like we do in most other widgets.

Here's and example: https://github.com/JoinColony/colonyDapp/blob/feature/dev-358-rewards/src/modules/dashboard/components/ActionsPage/FinalizeMotionAndClaimWidget/FinalizeMotionAndClaimWidget.tsx#L202-L216